packages/runtime/plugin-runtime/tests/document/index.test.tsx

```diff
@@ -92,4 +92,29 @@ describe('plugin-document', () => {
 
     expect(docHtml.includes(`<h1>title: </h1><div id="abc">`)).toBeTruthy();
   });
+
+  it('should render title successful', () => {
```

Comment:
May this test can be add to `tests/integration/app-document`, to make sure the html.title worked

Reply:
I will update this later

Reply:
done